src/shared/locales/enUs/translations/user/index.ts

```diff
@@ -13,6 +13,7 @@ export default {
           required: 'Password is required.',
         },
       },
+      success: 'Currently signed in as ',
```

Comment:
Let's use `currentlySignedInAs` as the key instead of `success`. I think that it makes it more clear what the key is supposed to represent.

Reply:
Let's leave off the trailing space in the key and make whoever uses this key put a space in.